src/data/CoreData.tsx

```diff
@@ -116,6 +121,7 @@ export function useCoreData(): {
     connectWritingSystems(languagesBySchema.Inclusive, territories, writingSystems);
     connectLocales(languagesBySchema.Inclusive, territories, writingSystems, locales);
     createRegionalLocales(territories, locales); // create them after connecting them
+    addIANAVariantLocales(languagesBySchema, locales, ianaVariants);
```

Comment:
lgtm, you can simplify it a little bit by passing in the CLDR language list.
```suggestion
    addIANAVariantLocales(languagesBySchema.CLDR, locales, ianaVariants);
```